tools/changelog.py

```diff
@@ -76,9 +76,18 @@ def normalize_title(title):
     See the test suite for the potential matches.
     GitHub sends us a unicode string.
     """
-    if '🚀' in title:
-        title = title.replace('🚀', '')
-        title = title.strip()
+    # if '🚀' in title:
```

Comment:
Greenkeeper is going away next month (see https://greenkeeper.io/), so we can probably just delete this, rather than comment it out. Same for the corresponding tests.

Reply:
Yeah. Let's delete all greenkeeper related code.

Reply:
> Greenkeeper is going away next month (see https://greenkeeper.io/), so we can probably just delete this, rather than comment it out. Same for the corresponding tests.

Roger that.